resources/views/audubon-society.blade.php

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Audubon Society Capture Form</title>
+    </head>
+    <body style='text-align: center; border-width:5px; border-style:solid; color:black; width:fit-content'>
```

Comment:
It's fine to add styles in this test project, but very rarely should we add inline styles.  style="display: none" is commonly used if we want to hide by default.  That inline style is fine, but others shouldn't be used for the most part.